README.md

```diff
@@ -38,7 +38,23 @@ There's a preview version currently available at Microsoft Store.
 
 WIP
 
-## Download latest binaries
+## Downloads
+
+You may download the latest release for your system:
+
+### Windows binaries
```

Comment:
I think in this case it is called **Windows installer**, not the _**binaries**_

Reply:
Done